src/topology/algebra/ordered.lean

```diff
@@ -42,14 +41,20 @@ continuous_iff_is_closed.mp (hf.prod_mk hg) _ t.is_closed_le'
 lemma is_closed_le' (a : α) : is_closed {b | b ≤ a} :=
 is_closed_le continuous_id continuous_const
 
+lemma is_closed_Iic {a : α} : is_closed (Iic a) :=
```

Comment:
```suggestion
lemma is_closed_Iic (a : α) : is_closed (Iic a) :=
```
Or is there a reason to prefer implicit here?

Reply:
I am just following the conventions in this file. For instance in `lemma is_open_Ioo {a b : α} : is_open (Ioo a b)`. In practice, when using this lemma I never needed to specify the parameter, so it seems all right like this.

Reply:
I was comparing to `is_closed_le'` above and `is_closed_ge'` below, which have `a` explicit. But you're right, at least for the uses in this PR, implicit seems fine.